### DISCUSSION TITLE
'lsp-complete' positions cursor at the beginning of the completion after candidate has been selected.

### DISCUSSION THREAD

**Discussion by bobdobbs** [emacs-lsp/lsp-mode#4161] [Q&A]
I'm using lsp-mode with rust-mode in emacs 2.9.
I'm seeing this behaviour in other major modes as well.

After a candidate has been selected, the cursor goes to the beginning of the newly-created completion.

This contrasts with other completion packages, which positions the cursor at the end of the completion.

So in the case of lsp-completion-mode, an extra two keystrokes are required after the completion.

At this point I don't know if this is the default behaviour of lsp-completion-mode, if it's configurable or if something in my configuration is interfering with a more desireable behavour.

**Comment by yyoncho** [comment]
Can you show a gif? As a side note, lsp-mode provides a backend for the completion frontends. The issue that you are describing sounds more like a frontend issue.
